UnityProject/Assets/Scripts/Core/Transform/CustomNetTransform.cs

```diff
@@ -144,6 +146,13 @@ private MotionStateEnum MotionState
 	public TransformState PredictedState => predictedState;
 	public TransformState ClientState => clientState;
 
+	private void Awake()
+	{
+		registerTile = GetComponent<RegisterTile>();
+		itemAttributes = GetComponent<ItemAttributesV2>();
+		syncInterval = 0f;
```

Comment:
does this need to be manually set on stuff?

Reply:
syncInterval = 0f;

Reply:
it would be if i didnt put that, making it 0 means it'll send the update immediately if the value was changed